inject-java-helper/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id "io.micronaut.build.internal.convention-library"
```

Comment:
We don't want to publish this, should be `id "io.micronaut.build.internal.convention-test-library"`